### PR TITLE
document MOLT replication (Replicator) flags

### PR DESCRIPTION
**Pull request by taroface**
DOC-11736

### CODE REVIEW

**Review by ryanluu12345 — APPROVED** [review]
Generally looks good to me, just a few nit comments. Deferring to @Jeremyyang920 for a last look here too

File: src/current/_includes/molt/replicator-flags.md
Comment (rohan-joshi) [inline src/current/_includes/molt/replicator-flags.md:1]: lgtm, thank you ryan

**Review by rohan-joshi — APPROVED** [review]

File: src/current/_includes/molt/replicator-flags.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+| `--slotName`        | `STRING`   | The replication slot in the source database.<br><br>**Default:** `"replicator"`   |
+| `--standbyTimeout`  | `DURATION` | How often to report WAL progress to the source server.<br><br>**Default:** `5s` |
+
+##### MySQL replication flags
+
+The following flags are set with [`--replicator-flags`](#global-flags) and can be used in any [Fetch mode](#fetch-mode) that involves replication from a [MySQL source database](#source-and-target-databases).
+
+|           Flag           |   Type   |                                               Description                                               |
+|--------------------------|----------|---------------------------------------------------------------------------------------------------------|
+| `--defaultGTIDSet`       | `STRING` | Default GTID set, in the format `source_uuid:min(interval_start)-max(interval_end)`.                    |
```
[the remark this answers is not in the document]
Comment (taroface) [reply src/current/_includes/molt/replicator-flags.md:60]: Doesn't your change to the replication mode behavior change this slightly? It sounds like this is only required the first time using `--replication-only`?

[the remark this answers is not in the document]
Comment (ryanluu12345) [reply src/current/_includes/molt/replicator-flags.md:60]: Yep, good point. This is only required for first time if you haven't run `--replication-only` or replicator at all before. But it's still something you can technically override any time -- let's say if you want to use an older GTID than is stored (not sure why you'd want to do that, but we support it).

Either way, good to mention that this is important to pass in if you haven't run replication yet and need to start from a starting point you define or know about.

File: src/current/_includes/molt/replicator-flags.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+| `--logFormat`                | `STRING`   | Choose log output format: `"fluent"`, `"text"`.<br><br>**Default:** `"text"`                                                                  |
+| `--metricsAddr`              | `STRING`   | A `host:port` on which to serve metrics and diagnostics.                                                                                      |
+| `--parallelism`              | `INT`      | The number of concurrent database transactions to use.<br><br>**Default:** `16`                                                               |
+| `--quiescentPeriod`          | `DURATION` | How often to retry deferred mutations.<br><br>**Default:** `10s`                                                                              |
+| `--retireOffset`             | `DURATION` | How long to delay removal of applied mutations.<br><br>**Default:** `24h0m0s`                                                                 |
+| `--scanSize`                 | `INT`      | The number of rows to retrieve from staging.<br><br>**Default:** `10000`                                                                      |
```
Comment (florence-crl) [inline src/current/_includes/molt/replicator-flags.md:18]: Reading down the alphabetical list of flags, I did not know what "staging" is referring to? After going down the list, I assume it is referring to `--stagingSchema`.  Is this a correct assumption?

File: src/current/_includes/molt/replicator-flags.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+| `--schemaRefresh`            | `DURATION` | How often a watcher will refresh its schema. If this value is zero or negative, refresh behavior will be disabled.<br><br>**Default:** `1m0s` |
+| `--sourceConn`               | `STRING`   | The source database's connection string.                                                                                                      |
+| `--stageMarkAppliedLimit`    | `INT`      | Limit the number of mutations to be marked applied in a single statement.<br><br>**Default:** `100000`                                        |
+| `--stageSanityCheckPeriod`   | `DURATION` | How often to validate staging table apply order (`-1` to disable).<br><br>**Default:** `10m0s`                                                |
```
Comment (florence-crl) [inline src/current/_includes/molt/replicator-flags.md:22]: Again, I was not sure what "staging table" referred to, until I read down the list to `--stagingSchema`.

File: src/current/_includes/molt/replicator-flags.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+| `--stageSanityCheckWindow`   | `DURATION` | How far back to look when validating staging table apply order.<br><br>**Default:** `1h0m0s`                                                  |
+| `--stageUnappliedPeriod`     | `DURATION` | How often to report the number of unapplied mutations in staging tables (`-1` to disable).<br><br>**Default:** `1m0s`                         |
+| `--stagingConn`              | `STRING`   | The staging database's connection string.                                                                                                     |
```
Comment (florence-crl) [inline src/current/_includes/molt/replicator-flags.md:25]: Again, I was not sure what "staging database" referred to, until I read down the list to `--stagingSchema`.

**Review by florence-crl — APPROVED** [review]
lgtm with a nit about clarity around the term "staging"

File: src/current/_includes/molt/replicator-flags.md (hunk 2 above)
Comment (taroface) [reply src/current/_includes/molt/replicator-flags.md]: I clarified this description as it was 1) the least vague; 2) the first instance of "staging" in the table. I will do more work to contextualize "staging" in follow-up PRs that integrate Replicator into the MOLT Fetch docs.

**Pull request merged**